src/manifestoo_core/git_postversion.py

```diff
@@ -57,9 +57,7 @@ def _run_git_command_lines(
 
 
 def _is_git_controlled(path: Path) -> bool:
-    with Path(os.devnull).open("w") as devnull:
-        r = _run_git_command_exit_code(["rev-parse"], cwd=path, stderr=devnull)
-        return r == 0
+    return path.joinpath(".git").exists()
```

Comment:
```suggestion
    return path.joinpath(".git").is_dir()
```

Reply:
If you use git worktree, it won't be a dir, so the current oode is more accurate.